validate/files.py

```diff
@@ -1,26 +1,14 @@
 ''' File checking functions '''
 from pathlib import Path
-import os
 import warnings
 
 def check_compressed(path:Path, file_extension:str):
     ''' Check file is compressed '''
-    if not file_extension.endswith('.gz'):
-        warnings.warn(f'Warning: file {path} is not zipped.')
+    compression_extensions = ['.gz']
+    if not any(file_extension.endswith(ext) for ext in compression_extensions):
```

Comment:
Should this function get the extensions from the `path` variable instead of having a separate `file_extension` parameter?

Reply:
We could also do that but since the extension is already pulled from `path` anyways; when we switch to `python-magic` to check compression, we won't need the extension for checking compression